colors/nord.vim

```diff
@@ -160,6 +160,7 @@ call s:hi("healthWarning", s:nord13_gui, s:nord1_gui, s:nord13_term, s:nord1_ter
 
 "+- Neovim Terminal Colors -+
 if has('nvim')
+  call s:hi("TermCursorNC", "", s:nord1_gui, "", s:nord1_term, "", "")
```

Comment:
Can you please move the new line out of the `if` statement body and append it to the *"+- Neovim Support -+* section above?